browser/resources/settings/email_aliases_page/content/email_aliases_modal.tsx

```diff
@@ -146,25 +147,25 @@ export const EmailAliasModal = (
     }
 ) => {
   const [limitReached, setLimitReached] = React.useState<boolean>(false)
-  const [proposedAlias, setProposedAlias] = React.useState<string>(
-    editState?.alias?.email ?? '')
   const [proposedNote, setProposedNote] = React.useState<string>(
     editState?.alias?.note ?? '')
   const [awaitingProposedAlias, setAwaitingProposedAlias] =
     React.useState<boolean>(true)
+  const [generateAliasResult, setGenerateAliasResult] =
+    React.useState<GenerateAliasResult | null>(null)
```

Comment:
this is completely a matter of personal preference so up to you but I'd normally type this as:

```tsx
const [generateAliasResult, setGenerateAliasResult] =
    React.useState<GenerateAliasResult>()

// generateAliasResult is `GenerateAliasResult | undefined`
```
and then use `undefined` instead of `null`

up to you :smile:

Reply:
works for me; fixed